UPGRADING.md

```diff
@@ -85,6 +85,22 @@ Additionally, a simplification of the start command leads to the following chang
 
 #### Server (`app.go`)
 
+##### TX Decoder
+
+In order to support x/accounts properly we need to init a `TxDecoder`:
```

Comment:
This should be a note under x/accounts upgrading.md, it is too high right now imho.
Especially because this only necessary for non depinject chains (we should preceise that too)